src/cli.rs

```diff
@@ -68,6 +71,20 @@ impl Rover {
             .get(RoverEnvKey::Home)?
             .map(|p| PathBuf::from(&p)))
     }
+
+    pub(crate) fn get_git_context(&self) -> GitContext {
+        // use these env vars as overrides to GitContext creation
+        let branch = self.env_store.get(RoverEnvKey::GitBranch).unwrap();
+        let committer = self.env_store.get(RoverEnvKey::GitCommitter).unwrap();
+        let commit = self.env_store.get(RoverEnvKey::GitCommit).unwrap();
+        // this url is still dirty, make sure not to use it anywhere else :)
+        let remote_url = self.env_store.get(RoverEnvKey::GitRemoteUrl).unwrap();
```

Comment:
these should all be `?`

Reply:
Hehe riiiiiiight

Reply:
Oh wait, no. Because I don't want to return from the fn if one of those aren't there

Reply:
But I also don't want panics 🤔 let me fixxxxxx

Reply:
Got it! `unwrap_or` does the trick :)